modules/oc-mirror-creating-image-set-config.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected.adoc
+
+:_content-type: PROCEDURE
+[id="oc-mirror-creating-image-set-config_{context}"]
+= Creating the image set configuration
+
+Before you can use the oc-mirror plug-in to mirror image sets, you must create an image set configuration file. This image set configuration file defines which {product-title} releases, Operators, and other images to mirror, along with other configuration settings for the oc-mirror plug-in.
+
+You are required to specify a storage back end in the image set configuration file. This storage back end can be a local directory or a docker v2 registry. The oc-mirror plug-in stores metadata in this storage back end during image set creation.
+
+[IMPORTANT]
+====
+Do not delete or modify the metadata that is generated by the oc-mirror plug-in executions. You must use the same storage back end on subsequent executions of oc-mirror to the same mirror registry.
```

Comment:
@bergerhoffer In this message about the metadata or maybe in a separate IMPORTANT message, we need to alert users that if they specify a top-level namespace in their initial run of `oc-mirror` they cannot change it on subsequent runs.
Ex.
```
./bin/oc-mirror --config ~/configs/imageset-config.yaml docker://localhost:5000/mynamespace
```
`mynamespace` cannot change

Reply:
Thanks @jpower432! I make this mention a few places if you can see what you think. Diff of my changes is here: https://github.com/openshift/openshift-docs/compare/b726ac339e22ff9dc29e653cafc280d54ecb7a50..79be0b985d976af7506281c84045ecb3ef73c7dc

Reply:
Looks good. Thanks!